### PR TITLE
Bugfix/purchase credits

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** Issue #91 

**This PR will result in the following new package version:** `v0.9.1`

This is only making a calculation change to the amount totals within the purchase intermediate models. As such, this will not be a breaking change for existing users.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

This PR adjusts the `amount` and `total_amount` calculations within the `int_quickbooks__purchase_transactions` model. In particular this PR takes into consideration the fact that certain purchases can be credits (for credit card only transactions). When these purchases are `true` for a credit we should be subtracting them from the total. As such, the adjustment in this PR ensures we are subtracting the total when the purchases are credits.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt compile
- [X] dbt run –full-refresh
- [X] dbt run
- [X] dbt test
- [n/a] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

I was able to confirm that for all purchase transactions that are `credit` type `true` were showing as a negative value and will thus be subtracted from the total. Additionally the user within Issue #91 confirmed the totals are now summing to the expected value. As such, I am reasonably confident this change is validated and appropriate for rollout.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [n/a] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [X] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🤑

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.9.1
+[PR #93](https://github.com/fivetran/dbt_quickbooks/pull/93) includes the following updates:
```
Comment (fivetran-avinash) [inline CHANGELOG.md:2]: Should we move to this to the end of line 4, as we do with other PRs in the CHANGELOG?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+## Bug Fixes
+- Adjusted the purchase amount totals within the `int_quickbooks__purchase_transactions` model to factor in credits when calculating purchase amounts.
```
Comment (fivetran-avinash) [inline CHANGELOG.md:4]: Add an empty line to line 5 to separate out the release notes

**Review by fivetran-avinash — APPROVED** [review]
Tested and works! Some small changes to the CHANGELOG recommended, but good to go.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:2]: We have typically only be including the end of line PR reference when there are multiple PRs in a single CHANGELOG. When it is only one, this approach should be appropriate.

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:4]: Updated.

**Pull request merged**